tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py3{8,9,10,11,12}, mypy, black, pylint
+isolated_build = True
+skip_missing_interpreters = True
+
+[testenv]
+deps = pytest
+commands = pytest
+
+[testenv:black]
+deps = black
+skip_install = True
+commands = black --check --diff sphinxlint/ tests/
+
+[testenv:mypy]
+deps =
+  mypy
+  types-polib
+skip_install = True
+commands = mypy --exclude fixtures --ignore-missing-imports sphinxlint/ tests/
+
+[testenv:pylint]
+deps =
+  pylint
+  pytest
```

Comment:
Is pytest needed here? 

Can we skip_install?

Reply:
Yes pytest is "needed" just so pylint can find the import to check we're using it correctly.

Reply:
Same for `skip_install`: without it we won't have `regex`, `polib`, ... and pylint will complain about not being able to import them.